proof_iavl_absence.go

```diff
@@ -0,0 +1,87 @@
+package iavl
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/crypto/merkle"
```

Comment:
If IAVL is going to remain its thing/own repo (which I think it should), it would really be nice to not take more dependency that couple it tightly to Tendermint at a particular version. Ultimately it would be good to drop dependencies on tendermint altogether if we can spin tmlibs/db out into its own thing see #46.

For `ProofOperator` you can just replicate the interface (or even better the subset of the interface you actually use) here - which makes it clearer what this package. For `ProofOp` could you do a bit of mapping elsewhere when you want to use the protofbuf type?

A little bit of duplication could make this library more lean and loosely coupled.

Reply:
I agree, but to do after game of stakes.